package.json

```diff
@@ -40,7 +40,7 @@
     "babel-plugin-transform-es2015-modules-amd": "^6.24.1",
     "bower-config": "^1.3.0",
     "bower-endpoint-parser": "0.2.2",
-    "broccoli": "^2.0.0-beta.3",
+    "broccoli": "^2.0.0-beta.4",
```

Comment:
I think we spoke, we should likely cut stable so that we don't need to rely on beta's. Doing so is likely a pre-req to enabling this feature by default.